submitter/relayer/relayer.go

```diff
@@ -295,14 +301,21 @@ func (rl *Relayer) convertCkptToTwoTxAndSubmit(ckpt *ckpttypes.RawCheckpointResp
 func (rl *Relayer) ChainTwoTxAndSend(data1 []byte, data2 []byte) (*types.BtcTxInfo, *types.BtcTxInfo, error) {
 	// recipient is a change address that all the
 	// remaining balance of the utxo is sent to
-	tx1, err := rl.buildTxWithData(data1, nil)
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to add data to tx1: %w", err)
-	}
+	tx1 := rl.lastSubmittedCheckpoint.Tx1
+	// prevent resending tx1 if it was successful
+	if rl.lastSubmittedCheckpoint.Tx1 == nil {
```

Comment:
Relevant changes here, cache the TX1 if successful

Reply:
is there any chance for `rl.lastSubmittedCheckpoint.Tx1` be loaded with the tx1 sent from the previous epoch?